core/src/main/java/com/sequenceiq/cloudbreak/service/upgrade/rds/RdsUpgradeOrchestratorService.java

```diff
@@ -88,19 +89,13 @@ public void validateDbDirectorySpace(Long stackId) throws CloudbreakOrchestrator
                 stateParams.getTargetHostNames(), "df -k /dbfs | tail -1 | awk '{print $4}'");
         Map<String, String> dataDirSizeMap = hostOrchestrator.runCommandOnHosts(List.of(stateParams.getPrimaryGatewayConfig()),
                 stateParams.getTargetHostNames(), "du -sk /dbfs/pgsql | awk '{print $1}'");
-        Integer dataDirSize = dataDirSizeMap.entrySet().stream()
```

Comment:
These changes seem unrelated to the rest of PR

Reply:
it's an optimization :) because I had to understand the code, and I did the refactor. the result is exactly the same